core/src/components/modal/usage/react.md

```diff
@@ -153,3 +153,47 @@ In most scenarios, setting a ref on `IonRouterOutlet` and passing that ref's `cu
   <IonButton onClick={() => setShow2ndModal(false)}>Close Modal</IonButton>
 </IonModal>
 ```
+
+
+### Sheet Modals
+
+```tsx
+const App: React.FC = () => {
+  const routerRef = useRef<HTMLIonRouterOutletElement | null>(null);
+  
+  return (
+    <IonApp>
+      <IonReactRouter>
+        <IonRouterOutlet ref={routerRef}>
+          <Route path="/home" render={() => <Home router={routerRef.current} />}  exact={true} />
+        </IonRouterOutlet>
+      </IonReactRouter>
+    </IonApp>
+  )
+};
+
+...
+
+interface HomePageProps {
+  router: HTMLIonRouterOutletElement | null;
+}
```

Comment:
Is this content needed to explain the usage of `ion-modal`? The shorter the code block the better.